<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGUI</class>
 <widget class="QDialog" name="DialogGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="zigbee_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="zig_close_pushButton">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>106</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/请1.png);</string>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>106</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/驴1.png);</string>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>162</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/72fca40728e3c2cdd43f7c95.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>zig_close_pushButton</zorder>
  <zorder>refresh_pushButton</zorder>
  <zorder>zigbee_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
